### ISSUE TITLE
Detail about the dataset and hyper-parameter

### ISSUE THREAD

**Issue by hosytuyen** [SystemErrorWang/CartoonGAN#8]
Hi, Thank you for amazing CartoonGAN implementation,
I'm reproducing CartoonGAN, but don't get the good result. 
Can your share how many real and cartoon images in your dataset. And during the training,  the weight of vgg loss, learning rate and batch_size that you using. Is it like in the your old_code folder?
Thanks you so much.

**Comment by SystemErrorWang** [comment]
In fact the model in this repo is not trained with the same model described in CartoonGAN paper. I am sorry that the training code could not be published now, but after I subscrib my paper I may open source them.

**Comment by hosytuyen** [comment]
Many thanks for your reply, I look forward to your paper.

**Comment by SystemErrorWang** [comment]
> Many thanks for your reply, I look forward to your paper.

for dataset, due to copyright issues i may not be able to share, but i can give you some advice:
For human face: ffhq dataset or celeba hq(resize to 256*256)
For animate face: kyoto animate and pa works, use animate face detection and manually clean them
For real world photo: CycleGAN training dataset
For animate image: Shinkai Makoto, Hosoda Mamoru, Miyazaki Hayao, get key frames and random crop to 256*256 (if you manually clean them you may get better result)

**Comment by SystemErrorWang** [comment]
@HoSyTuyen  Paper is accepted to CVPR2020, code and other material will be open-sourced later

**Comment by hosytuyen** [comment]
Wao, Congratulates!!!, It's very impressive, thanks for your sharing.
